xarray/util/generate_aggregations.py

```diff
@@ -22,12 +22,13 @@
 
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Any, Callable, Sequence
+from collections.abc import Sequence
+from typing import TYPE_CHECKING, Any, Callable
```

Comment:
This change was supposed to be a separate commit, but this was only because ruff raised an error after regenerating the `_aggregations.py` file.